cogs/audio.py

```diff
@@ -2210,6 +2322,36 @@ def has_connect_perm(self, author, server):
                     message = escape(message, mass_mentions=True)
                     await self.bot.send_message(next_channel, message)
 
+    async def disp_now_playing(self, server, song, notifychan:int):
+        channel = discord.utils.get(server.channels, id=notifychan)
+        if channel != None:
```

Comment:
I would short circuit (aka return early) this `if` and some of the following ones to reduce the nested indentation level. Also remember that with `True`, `False` and `None` `is` / `is not` must be used instead.